controllers/storage.go

```diff
@@ -79,6 +79,7 @@ func (s *Storage) ArtifactFor(kind string, metadata metav1.Object, fileName, rev
 		Path:           localPath,
 		URL:            url,
 		Revision:       revision,
+		Checksum:       checksum,
 		LastUpdateTime: metav1.Now(),
```

Comment:
I think we should change this in a future iteration to the creation timestamp of the file, and not the creation time of the object.